Documentation/contributing/development/reviewers_committers/review_vendor.rst

```diff
@@ -0,0 +1,117 @@
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _review_vendor:
+
+****************************
+Reviewing for @cilium/vendor
+****************************
+
+What is @cilium/vendor?
+=======================
+
+Team `@cilium/vendor <vendor_team_>`_ is a GitHub team of Cilium contributors
+who are responsible for maintaining the good state of Go dependencies for
+Cilium and it's related projects by reviewing Pull Requests (PRs) that update
+files related to dependency declaration:
+
+* `go.mod <go_dot_mod_>`_
+* `go.sum <go_dot_sum_>`_
+* `vendor/ <vendor_slash_>`_
+
+Each time a non-draft PR touching files owned by the team opens, GitHub
+automatically assigns one member of the team for review. The team is currently
+responsible for reviewing changes related to Go dependencies in the following
+repositories:
+
+* `cilium/cilium <cilium_slash_cilium_reviews_>`_
+* `cilium/hubble <cilium_slash_hubble_reviews_>`_
+* `cilium/cilium-cli <cilium_slash_cilium_cli_reviews_>`_
+* `cilium/certgen <cilium_slash_certgen_reviews_>`_
+
+Open Pull Requests awaiting reviews from @cilium/vendor are listed by repository
+through the previously listed links.
+
+To join the team, you must be a Cilium Reviewer. see `Cilium's Contributor
+Ladder <ladder_>`_ for details on the requirements and the application process.
+
+.. _vendor_team: https://github.com/orgs/cilium/teams/vendor
+.. _go_dot_mod: https://github.com/cilium/cilium/blob/main/go.mod
+.. _go_dot_sum: https://github.com/cilium/cilium/blob/main/go.sum
+.. _vendor_slash: https://github.com/cilium/cilium/blob/main/vendor
+.. _cilium_slash_cilium_reviews: https://github.com/cilium/cilium/pulls?q=is%3Apr+is%3Aopen+draft%3Afalse+team-review-requested%3Acilium%2Fvendor
+.. _cilium_slash_hubble_reviews: https://github.com/cilium/hubble/pulls?q=is%3Apr+is%3Aopen+draft%3Afalse+team-review-requested%3Acilium%2Fvendor
+.. _cilium_slash_cilium_cli_reviews: https://github.com/cilium/cilium-cli/pulls?q=is%3Apr+is%3Aopen+draft%3Afalse+team-review-requested%3Acilium%2Fvendor
+.. _cilium_slash_certgen_reviews: https://github.com/cilium/certgen/pulls?q=is%3Apr+is%3Aopen+draft%3Afalse+team-review-requested%3Acilium%2Fvendor
+.. _ladder: https://github.com/cilium/community/blob/main/CONTRIBUTOR-LADDER.md
+
+Reviewing Pull Requests
+=======================
+
+This section describes some of the process and expectations for reviewing PRs
+on behalf of @cilium/vendor. Note that :ref:`the generic PR review
+process for Committers <review_process>` still applies, even though it is not
+specific to dependencies.
+
+Existing Dependencies
+---------------------
+
+Updates to existing dependencies most commonly occur through PRs opened by
+`Renovate <renovate_>`_, which is a 3rd party service used throughout the
+Cilium organization. Renovate continually checks repositories for out-of-date
+dependencies and opens new PRs to update any it finds.
+
+.. note::
```

Comment:
I bring this up to try to level-set expectations around using these call-outs - most commonly I notice PRs which will add a note because the contributor does not know how to integrate the text into the page, so it's easier for them to just add a call-out note and not consider how the overall doc flows.... this is not really the case here as obviously you are introducing a new page and as a single author you can make the entire thing flow well.

That said, I wonder if this exposition is actually trying to address a lack of documentation elsewhere to point towards the renovate configuration. I'm not sure it necessarily fits here, but I won't quibble about that. Though re-reading this paragraph I don't quite understand what it is trying to communicate to the target audience - vendor reviewers. "Here is where to find the renovate configuration, but you shouldn't need to worry about it"? There is also the option to just omit this particular piece of information; unless we anticipate that vendor reviewers will need to modify renovate configuration on a regular basis, then it may not need to live here at all.

Reply:
(Overall though this is really a nit and doesn't detract from the PR which I very much appreciate :pray: )

Reply:
Ah interesting. Thank you for explaining your perspective here. I always treated the note callout as a side-piece of non-essential information, but I see what you mean about it being a highlight. I think that my goal with the paragraph was to explain the "magic" behind the renovate bot, which new members of the renovate team will be spending a decent amount of time interacting with. Taking a step back, I think there is a bit of a gray area for who is "assigned" the responsibility of managing the renovate configuration since it is consumed by a couple of different teams.

This really resonated with me:

> I wonder if this exposition is actually trying to address a lack of documentation elsewhere to point towards the renovate configuration. 

When I initially wrote this paragraph, I started the first sentence with "For the curious", which I think shows how it is non-essential to this page, but the information may still be useful for those who need to dig deeper with renovate. I'll take it out here, and maybe we could follow up with more documentation for renovate at a later point.